core/lib/spree/permission_sets/default_customer.rb

```diff
@@ -4,11 +4,11 @@ module Spree
   module PermissionSets
     class DefaultCustomer < PermissionSets::Base
       def activate!
-        can :display, Country
-        can :display, OptionType
-        can :display, OptionValue
+        can :read, Country
+        can :read, OptionType
+        can :read, OptionValue
         can :create, Order
-        can [:read, :update], Order do |order, token|
+        can [:show, :edit, :update], Order do |order, token|
```

Comment:
block conditions should be defined only for actions on object instances, not classes, therefore this rule applies only on `:show, :edit, :update`. This was actually working by mistake since version 2.0 of cacancan